website/content/docs/agent/wal-logstore/index.mdx

```diff
@@ -11,6 +11,8 @@ This topic provides an overview of the experimental WAL (write-ahead log) LogSto
 
 !> **Experimental feature:** The WAL LogStore backend is experimental.
 
+!> **Known Issues:** Consul 1.15.0 and 1.15.1 have [known issues](#known-issues) so should not have WAL enabled in production.
```

Comment:
```suggestion
!> **Known Issues:** Consul 1.15.0 and 1.15.1 have [known issues](#known-issues) so should not have WAL enabled in production.
```
We should move this to the usage pages associated with this functionality: https://developer.hashicorp.com/consul/docs/agent/wal-logstore/enable#requirements

Also, this callout and the *Experimental features* callout are inconsistent with our style guidance on usage: https://github.com/hashicorp/engineering-docs/blob/main/writing/style-guide.md#callout-boxes

Reply:
I agree that consecutive callouts isn't ideal but conveying both important notes in one didn't seem ideal either. I'm super open to alternative suggestions though!

I also considered putting this callout on the enable page too. I settled on here because this is the page we be most likely to link people to and until we have a fix I don't want to "bury the lead" that they shouldn't be using this yet several pages into their investigation.

Reply:
The topic description sentence says that the feature is experimental, so I think the first callout is unnecessary and actually distracts the reader--especially right at the top of the page. The same goes for the second callout. 

My suggestion is that we get rid of that callouts and revise the introductory statement:

"This topic provides an overview of the WAL (write-ahead log) LogStore backend. The WAL backend is an experimental feature. Refer to [Requirements]([/consul/docs/agent/wal-logstore/enable#requirements](https://developer.hashicorp.com/consul/docs/agent/wal-logstore/enable#requirements)) for supported environments and known issues. "

We just need to be super judicious with how and when we use callouts. Most style guides consider them distractions but allow them only when absolutely necessary.

Reply:
Thanks I'll update as you proposed with some additional clarity on the level of risk being taken.

Personally, I'd always err on the side of being as obvious as possible when describing a feature that is not ready for production usage yet so there is no room for missing that if people are skimming. It's _meant_ to be a distraction in the sense that no reader should be able to look at the page and be in doubt about whether this is ready to use in production or not 😄 But I also get why two callouts together reduces the impact so I'll change it to match your suggestion. "Experimental" in the title is a clue, but there is "experimental" as in "the API might change one day" and "experimental" as in "you may loose all your data and be unable to recover" which is worth being clear about IMO!